src/bin/git-stack/stack.rs

```diff
@@ -1469,7 +1469,7 @@ const SPACE_GLYPHS: termtree::GlyphPalette = termtree::GlyphPalette {
 };
 
 const JOINT_GLYPHS: termtree::GlyphPalette = termtree::GlyphPalette {
-    item_indent: "─┐",
+    item_indent: "─╮",
```

Comment:
How universally does this work?  In my browser at least, the lines don't line up

Reply:
it is part of the unicode "box drawing characters" block: https://www.fileformat.info/info/unicode/block/box_drawing/fontsupport.htm

It works well enough for `tig` that nobody seems to have complained about it so far: https://github.com/jonas/tig/blob/06a1b89d98f32e9ac2d81b94515b200b0acb7dbb/src/graph-v2.c#L1081

Reply:
also it is part of the default ratatui charset https://docs.rs/ratatui/latest/src/ratatui/symbols.rs.html#107